### DISCUSSION TITLE
Mozilla's compass on AI broken or just a communication stance? (Mozilla AI not involved?)

### DISCUSSION THREAD

**Discussion by SphinxKnight** [mdn/mdn-community#420] [Announcements]
I'm struggling to see a clear strategy inside Mozilla regarding AI and its flaws.

This is from March this year: https://blog.mozilla.org/en/mozilla/introducing-mozilla-ai-investing-in-trustworthy-ai/

Emphasis on:

"Mozilla has been asking these questions about AI for a while now —[ sketching](https://blog.mozilla.org/en/mozilla/mozillas-vision-for-trustworthy-ai/) out a vision for trustworthy AI,[ mobilizing](https://foundation.mozilla.org/en/blog/regretsreporter-user-controls-study-1/) our community to document what’s broken" 

"if we design the technology very differently — if we put human agency and the interests of users at the core, and if we prioritize transparency and accountability. " (see https://github.com/orgs/mdn/discussions/419) 

"The less good news: We don’t see this happening amongst the big tech and cloud companies with the most power and influence. Meanwhile, these incumbents continue to consolidate their control over the market." 

I'm seeing no mention of Mozilla AI in https://developer.mozilla.org/en-US/blog/introducing-ai-help/ nor https://blog.mozilla.org/en/products/mdn/responsibly-empowering-developers-with-ai-on-mdn/ 

The two directions (esp when looking at https://github.com/mdn/yari/issues/9208) seems quite contradictory.

1. What is the *effective* stance of Mozilla on AI now? (and I mean Mozilla Corporation *and* Mozilla Foundation, do not expect the outside world to accept a "paradox" there)

2. Has there been any meeting between Steve Teixeira's team (I had never seen his name in an MDN context before by the way, maybe an introduction would have been helpful) and Moez Draief's team? If not, why?

3. What is the answer from MDN's teams to each (quoted) concern? esp. now that https://github.com/mdn/yari/issues/9208 has been raised?

**Comment by caugner** [comment]
We have recorded your questions (left) with the following numbers (right):

1. ➡️ 37
2. ➡️ 38
3. ➡️ 39

**Comment by SphinxKnight** [comment]
https://foundation.mozilla.org/en/blog/chat-gpt-ai-seo-blog-post/ seems to indicate MoFo is not aligned with MoCo.
Chosen excerpts (echoing other discussions/questions regarding this topic)

": It’s worth considering that AI models providing “instant and accurate responses to queries” leads people to believe that they can trust the outputs of the model, when actually we've seen numerous examples of ChatGPT hallucinations. Overreliance is one of the most challenging risks in the adoption of large language models, which OpenAI themselves write about in [the documentation of the model](https://cdn.openai.com/papers/gpt-4-system-card.pdf). I wonder if ChatGPT has read its own documentation?"

"In actuality, ChatGPT was only made possible through the non-consensual and extractive use of data already available on the internet. Furthermore, there are many [restrictions](https://openai.com/policies/usage-policies) on how data generated by ChatGPT can be used."
